src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,80 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(toFileName);
+        file.delete();
+        String readFromFileString = readFromFile(fromFileName);
+        String[] makeReportArray = makeReport(readFromFileString);
+        writeToFile(makeReportArray, toFileName);
+    }
+
+    private String readFromFile(String fromFileName) {
+        StringBuilder stringBuilder = new StringBuilder();
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName));
+            int value = bufferedReader.read();
+            while (value != -1) {
+                stringBuilder.append((char) value);
+                value = bufferedReader.read();
+            }
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException(e);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+        return stringBuilder.toString();
+    }
+
+    private String[] makeReport(String input) {
+        int supply = 0;
+        int buy = 0;
+        int result = 0;
+        String[] inputArray = input.split("\n");
```

Comment:
You can simplify your logic 
```suggestion
        String[] splitWords = line.split(COMMA_SEPARATOR);
                if (splitWords[OPERATION_INDEX].equals(BUY_OPERATION)) {
                    buy += Integer.parseInt(splitWords[AMOUNT_INDEX]);
                } else {
                    supply += Integer.parseInt(splitWords[AMOUNT_INDEX]);
                }
```

where COMMA_SEPARATOR, BUY_OPERATION, OPERATION_INDEX, AMOUNT_INDEX are constant